sql/updates/world/master/9999_99_99_01_world.sql

```diff
@@ -0,0 +1,835 @@
+-- Halls of Origination work
+SET @CGUID := 1000000; -- 91 entries needed (1000000 - 1000090)
+SET @SpellEffectID := 1000000; -- 2 entries needed
+
+-- General: Elevator, trash mobs --
+-- spawn missing Temple Fireshaper (not in old sniffs)
+DELETE FROM `creature` WHERE `guid` = @CGUID+0;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `PhaseId`, `PhaseGroup`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `unit_flags2`, `unit_flags3`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(@CGUID+0, 48143, 644, 0, 0, 6, 0, 0, 0, 0, -640.624, 396.364, 83.8651, 1.54741, 7200, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, '', -1);
+
+-- script names, sniff corrections
+UPDATE `gameobject_template` SET `ScriptName` = 'go_hoo_the_makers_lift_controller' WHERE `entry` = 207669;
+UPDATE `creature_template` SET `ScriptName` = 'npc_hoo_aggro_stalker' WHERE `entry` = 40790;
+UPDATE `creature_template` SET `ScriptName` = 'npc_hoo_spatial_flux', `unit_flags` = 33554496 WHERE `entry` = 39612;
+UPDATE `creature_template` SET `ScriptName` = 'npc_hoo_energy_flux', `unit_flags` = 33554496, `speed_walk` = 5.5/2.5, `speed_run` = 5.5/2.5 WHERE `entry` = 44015;
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_hoo_emerge', 'spell_hoo_energy_flux_target_selector', 'spell_hoo_arcane_energy_check');
```

Comment:
cosmetic/codestyle: add a blank line between line 15 and 16 (different tables + query types)